src/cyhy_kevsync/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""The cyhy_kevsync library."""
+
+# We disable the following Flake8 checks:
+# - "Module level import not at top of file (E402)" here because the constants
+#   need to be defined early to prevent a circular import issue.
+# - "Module imported but unused (F401)" here because although this import is not
+#   directly used, it populates the value package_name.__version__, which is
+#   used to get version information about this Python package.
+
+DEFAULT_KEV_URL = "https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json"
+DEFAULT_KEV_SCHEMA_URL = "https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities_schema.json"
+
+from ._version import __version__  # noqa: F401, E402
+from .main import do_kev_sync  # noqa: E402
+
+__all__ = [DEFAULT_KEV_URL, DEFAULT_KEV_SCHEMA_URL, "do_kev_sync"]
```

Comment:
Sort alphabetically:
```suggestion
__all__ = [ "do_kev_sync", DEFAULT_KEV_SCHEMA_URL, DEFAULT_KEV_URL]
```

Reply:
Doh- my suggestion had an extraneous space character in it, which is [angering our linter](https://github.com/cisagov/cyhy-kevsync/actions/runs/11407433790/job/31743471426#step:21:54).  You should revert https://github.com/cisagov/cyhy-kevsync/pull/3/commits/fa716a00a168ca72dae3c87287f68bb034bae6d6 and then apply this corrected suggestion:
```suggestion
__all__ = ["do_kev_sync", DEFAULT_KEV_SCHEMA_URL, DEFAULT_KEV_URL]
```

Reply:
I see you fixed this in https://github.com/cisagov/cyhy-kevsync/pull/3/commits/b5aa8226d1222605af4f2404da4a4ebbafba233d. 👍